packages/lib/src/core/ProcessResponse/PaymentAction/actionTypes.ts

```diff
@@ -20,7 +20,16 @@ const getActionHandler = statusType => {
         const config = {
             ...props,
             ...action,
-            onComplete: props.onAdditionalDetails,
+            // onComplete should trigger `onAdditionalDetails` callback or `submitDetails` call for the sessions flow
```

Comment:
Can you expand this comment a bit to explain why the presence, or no, of `component` indicates that it's a /sessions flow